joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/controlflow/codepencegraph/CdgPass.scala

```diff
@@ -22,6 +22,8 @@ import org.slf4j.{Logger, LoggerFactory}
 class CdgPass(cpg: Cpg) extends ForkJoinParallelCpgPass[Method](cpg) {
   import CdgPass.logger
 
+  val hasLogged = java.util.concurrent.atomic.AtomicInteger(10)
```

Comment:
please add a comment why you chose 10..

Reply:
I don't particularly care about the precise number -- it's just that, if a graph is broken in a way that triggers this log message often, then it tends to trigger this message _very_ often, which takes forever (and spams the logs).

Reply:
Makes sense, and I was aware that 10 is just an arbitrary number, but whenever the next reader sees an arbitrary number they might wonder 'why that number', so a short comment is helpful IMO